pkg/cert/kube_certs.go

```diff
@@ -328,3 +313,82 @@ func (meta *MetaData) GenerateAll() error {
 	}
 	return nil
 }
+
+// GenerateCert generate all cert.
+func GenerateCert(certPath, certEtcdPath string, altNames []string, hostIP net.IP, hostName, serviceCIRD, DNSDomain string) error {
+	// if no certPath, certEtcdPath specified, will use default kubernetes pki path "/etc/kubernetes/pki".
+	if certPath == "" {
```

Comment:
Cert-related work is very very serious in the cluster operation. If a user has no input config, I suggest that we should block the request and must let them input clear and correct cert config.

In addition, I wish that in the command definition layer, we validate if the input cert config is empty.